lib/processors/newsprocessor.py

```diff
@@ -0,0 +1,147 @@
+from tornado import gen
+from .lib.utils import process_api_handler
+
+import re
+import random
+from .lib.baseprocessor import BaseProcessor
+
+
+class NewsProcessor(BaseProcessor):
+    name = 'news_processor'
+
+    def __init__(self):
+        super().__init__()
+
+    def fb_proxy(self, prof):
+        if 'political' in prof:
```

Comment:
do something similar for the other processors as well, but I'm a bit weary about returning magic number codes.